smtp.go

```diff
@@ -68,7 +68,7 @@ func NewClient(conn net.Conn, host string) (*Client, error) {
 		text.Close()
 		return nil, err
 	}
-	c := &Client{Text: text, conn: conn, serverName: host, localName: hostName}
+	c := &Client{Text: text, conn: conn, serverName: host, localName: "localhost"}
```

Comment:
Just wanting to double check since I don't have all the context here - is this a safe assumption to make?

Reply:
yeah - this brings the code back closer to what's in the stdlib `net/smtp` package where this was originally forked from. In practice we set this to `localhost` in our config anyway.

My hope is to eventually move back to the `net/smtp` package, so this is a step in that direction 😉

Reply:
Perfect! Thanks for the background :)